### ISSUE TITLE
types with type integer produces class with no body (not even pass)

### ISSUE THREAD

**Issue by jasco** [Jumpscale/go-raml#209]
The following raml

```
types:
  UserId:
    type:integer
```

produced a UserId.py (omitting imports):

```
class UserId(Form):

```

At a minimum it needs `pass` to be properly formed, though I am not sure, without a deeper look, whether this template is appropriate for simple types or not.

This is not currently a pressing issue.  I submit it in the hope that it helps the project.

**Comment by iwanbk** [comment]
```
This is not currently a pressing issue. I submit it in the hope that it helps the project.
```
Yes, it helps! thanks.

```
whether this template is appropriate for simple types or not.
```
Current go-raml only generate flask-wtf class for request body validation.

we will add support to generate python class from RAML types.
I have created issue for this in #210

**Comment by iwanbk** [comment]
PR for fix submitted at #234
